### ISSUE TITLE
[wok] make rpm fails on fedora 22

### ISSUE THREAD

**Issue by chandrureddy** [kimchi-project/wok#2]
Steps to reproduce:
1. ./autogen.sh --system
2. make
3. make rpm failed with the following error:
[chandra@localhost wok]$ make rpm
make  dist-gzip am__post_remove_distdir='@:'
make[1]: Entering directory '/home/chandra/wok'
fatal: No names found, cannot describe anything.
Makefile:1039: recipe for target 'VERSION' failed
make[1]: *** [VERSION] Error 128
make[1]: Leaving directory '/home/chandra/wok'
Makefile:708: recipe for target 'dist' failed
make: *** [dist] Error 2

For some weird reason VERSION file content will become empty after "make rpm" I guess.

**Comment by pvital** [comment]
I'm going to take a look on this issue.

**Comment by pvital** [comment]
I could execute 'make rpm' (following the same steps you did) with no problems on my Fedora 23 box:

```
Provides: wok = 1.5.1-0.fc23
Requires(interp): /bin/sh /bin/sh /bin/sh
Requires(rpmlib): rpmlib(CompressedFileNames) <= 3.0.4-1 rpmlib(FileDigests) <= 4.6.0-1 rpmlib(PartialHardlinkSets) <= 4.0.4-1 rpmlib(PayloadFilesHavePrefix) <= 4.0-1
Requires(post): /bin/sh systemd
Requires(preun): /bin/sh systemd
Requires(postun): /bin/sh systemd
Requires: /usr/bin/env python(abi) = 2.7
Checking for unpackaged file(s): /usr/lib/rpm/check-files /home/pvital/Projects/wok+all/rpm/BUILDROOT/wok-1.5.1-0.fc23.x86_64
Wrote: /home/pvital/Projects/wok+all/rpm/SRPMS/wok-1.5.1-0.fc23.src.rpm
Wrote: /home/pvital/Projects/wok+all/rpm/RPMS/noarch/wok-1.5.1-0.fc23.noarch.rpm
```

In addition, the content of VERSION file continues the same.

```
$ cat VERSION 
1.5.1
```

**Comment by chandrureddy** [comment]
May be I guess this issue could due to the Issue #1 raised earlier. So if the issue #1 solved then this will go away.

**Comment by alinefm** [comment]
Fixed by 910d1d81ca58cc9deddf0dac40fff48c6c8e1be6
